README.md

```diff
@@ -1,10 +1,10 @@
 SnoVault JSON-LD Database Framework
 ===================================
 
-## System Installation (OSX Catlina 10.15.2)
-    The instructions below are not guaranteed to work.  See the dependency's website for
-    furthur instruction.  We will try to keep this updated as OSX, Xcode, and brew update.  These
-    instructions were also tested on Mojave 10.14.6.
+## System Installation (OSX Catlina~10.15.2, Mojave~10.14.6)
```

Comment:
It is fine if you want to word it this way. But technically, these steps are guaranteed to work for the OS/packages version listed and not guaranteed for anything outside this scope.

Reply:
The lag between Us updating our README to match a dependency will always be greater than the Dependency updating it's readme.